app/src/main/java/com/example/lhi06/residue/AlbumReview.java

```diff
@@ -0,0 +1,44 @@
+package com.example.lhi06.residue;
+
+import android.support.annotation.NonNull;
+
+/**
+ * Created by lhi06 on 20/07/2017.
+ */
+
+public class AlbumReview {
+
+    private Album album;
+    private int rating;
+    private String review = "";
+
+    public AlbumReview(@NonNull final Album thisAlbum, final int rating, @NonNull final String review) {
+        this.album = thisAlbum;
+        this.rating = rating;
+        this.review = review;
+    }
+
+    public Album getThisAlbum() {
+        return album;
+    }
+
+    public void setThisAlbum(final Album thisAlbum) {
```

Comment:
remove setter until you need it - data objects are safer to use if they can't be modified after creation (i.e. they are immutable) so you should aim for this pattern where possible